addon/components/generic-rdfa-variable/insert-menu.ts

```diff
@@ -26,6 +26,14 @@ export default class GenericRdfaVariableInsertMenu extends Component<Args> {
     return this.args.controller.schema;
   }
 
+  get canInsert() {
```

Comment:
you technically don't need this check anymore now :sweat_smile: but it doesn't hurt